sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/mathExpressions.scala

```diff
@@ -804,7 +858,6 @@ case class Pow(left: Expression, right: Expression)
   }
 }
 
-
```

Comment:
Seems unrelated change. Could we just revert this back?

Reply:
Sure